nights/2020-02-19-README.md

```diff
@@ -1,39 +1,13 @@
 # 2020-02-19
 
-## Reviewed by:   your_name_here
+## Reviewed by:   Mara DeRung
 
 ## Raw image gallery: http://physics.mnstate.edu/feder_gallery/2020-02-19
 
 ## Unusual images?
 
 Are there any images that look unusual? List the file name of any unusual images for this night here, with description:
 
-+ `crazy-object-001R.fit` -- not sure what this is an image of
-+ `another-crazy-object-003B.fit` -- this looks more like a flat than a science image.
-+ `full-moon-005B.fit` -- nice satellite or plane track in here.
-
-Delete this list if there are no unusual images.
-
-## Missing information?
-
-Check these off if they are true:
-
-- [ ] No images are missing filter information (except BIAS and DARK, which need no filter).
-- [ ] No images are missing pointing information (RA/Dec and WCS)
-- [ ] No images are missing object names (only applies to science images)
-- [x] EXAMPLE checked-off box, please delete.
-
-If any images are missing information and you have been unable to fix them please list
-them below with a short description of the problem.
-
-+ `m34-002R.fit` -- no WCS, looks like there were maybe clouds.
-+ `m404-001B.fit` -- Not sure what object this, and googling `m404` got me a 404.
-+ `sa113-099V.fit` -- No filter in the FITS header, not sure what it should be.
-
-## What, if anything, did you have to do to fix images on this night?
-
-Remember, you should do your changes with scripts that you number and place in the
-directory along with the data so the procedure could be repeated if needed or
-desired.
-
-Here, explain in English (not code) what you fixed, if anything.
++ `S001-R001-C011 -- white  banding sections in  all the images some bands are brighter than others.
++ `ey-uma-S001-R001-C002,ey-uma-S001-R001-C017,-- satellite.
```

Comment:
Can you please add another backtick after the C017? You should be able to just accept the suggestion below.
```suggestion
+ `ey-uma-S001-R001-C002,ey-uma-S001-R001-C017`,-- satellite.
```